docs/docker-image-build.md

```diff
@@ -0,0 +1,29 @@
+# Build a `ks` docker image
+
+The intended use case for building a docker images is to run `ks` from CI/CD. As a development tool it is recommended to install `ks` locally, though it is still possible to use the docker image for development as well.
+
+Copy and paste the following commands:
+```bash
+# Clone the ksonnet repo into your GOPATH
+go get github.com/ksonnet/ksonnet
+
+# Build and install binary under shortname `ks` into $GOPATH/bin
+cd "${GOPATH}/src/github.com/ksonnet/ksonnet"
+make docker-image
+```
+
+# Running `ks` via docker
+
+In order to run via docker, the `ks` process needs access to a kubernetes config file, certificates, and the current working directory. We can set these up using mounts. Here's what it looks like running on a local cluster with minikube:
+
+```bash
+docker run -e KUBECONFIG="~/.kube/config" --mount type=bind,source="~/.kube/conifg",target="~/.kube/config" --mount type=bind,source="~/.minikube",target="~/.minikube" --mount type=bind,source="$(pwd)",target="$(pwd)" -w "$(pwd)" ks --help
+```
+
+This sets the $KUBECONFIG environment variable inside the container, mounts the config and the directory holding certificates (which can be found inside the kubeconfig), and mounts the current working directory so that the `ks` binary knows where to work.
+
+It is also possible to use as an alias:
```

Comment:
Clarify the intention of setting an alias. e.g:

Optionally, set an alias to shorten the verbose command if using the docker image locally.